webviewloader/module.go

```diff
@@ -10,15 +10,58 @@ import (
 )
 
 var (
-	nativeModule = windows.NewLazyDLL("WebView2Loader")
-	nativeCreate = nativeModule.NewProc("CreateCoreWebView2EnvironmentWithOptions")
+	nativeModule                 = windows.NewLazyDLL("WebView2Loader")
+	nativeCreate                 = nativeModule.NewProc("CreateCoreWebView2EnvironmentWithOptions")
+	nativeCompareBrowserVersions = nativeModule.NewProc("CompareBrowserVersions")
 
-	memOnce   sync.Once
-	memModule winloader.Module
-	memCreate winloader.Proc
-	memErr    error
+	memOnce                   sync.Once
+	memModule                 winloader.Module
+	memCreate                 winloader.Proc
+	memCompareBrowserVersions winloader.Proc
+	memErr                    error
 )
 
+// CompareBrowserVersions will compare the 2 given versions and return:
+//  -1 = v1 < v2
+//   0 = v1 == v2
+//   1 = v1 > v2
+func CompareBrowserVersions(v1 string, v2 string) (int, error) {
+	nativeErr := nativeModule.Load()
```

Comment:
I think it'd be cleaner to move this to after the UTF16PtrFromString calls, so it's next to the loadFromMemory version. It'd probably be even better if that part could be extracted so that both functions don't have to do that individually, but some stuff probably needs to be untangled for that to work out nicely.

Reply:
No worries. I noticed the types were slightly different (`uintptr` vs `uint64`) in the `Call` methods between native and memory. Is this unavoidable? It would greatly simplify the code if we could use an interface here.